packages/dashboard/src/components/Card/Card.types.ts

```diff
@@ -4,6 +4,7 @@ import { ICardContentDetails } from './Layout/Layout.types';
 import { IAction } from './ActionBar/ActionBar.types';
 import { ICardDropDownOption } from './CardFrame/CardFrame.types';
 import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IFocusZoneProps } from 'office-ui-fabric-react/lib/components/FocusZone/FocusZone.types';
```

Comment:
I'd recommend importing from the top-level unless you absolutely need this path import. I'm less familiar with the Dashboard package import pattern.